README.md

```diff
@@ -4,15 +4,15 @@
 
 ## Introduction
 
-A real-time application invariably discards any packet that arrives after the jitter buffer's play-out deadline. This applies for any real-time application, whether streamed video, interactive media or online gaming. For this broad class of applications a median delay metric is a distraction---waiting for the median delay before play-out would discard half the packets. To characterize the delay experienced by an application it would be more useful to quote the delay of a high percentile of packets. But which percentile? The 99th? The 99.99th? The 98th? The answer is application- and implementation-dependent, because it depends on how much discard can effectively be concealed (1%, 0.01% or 2% in the above examples, assuming no other losses). Nonetheless, it would be useful to settle on a single industry-wide percentile to characterize delay, even if it isn't perfect in every case.
+A real-time application invariably discards any packet that arrives after the jitter buffer's play-out deadline. This applies for any real-time application, whether streamed video, interactive media or online gaming. [Not all games use jitter buffers] For this broad class of applications a median delay metric is a distraction---waiting for the median delay before play-out would discard half the packets. To characterize the delay experienced by an application it would be more useful to quote the delay of a high percentile of packets. But which percentile? The 99th? The 99.99th? The 98th? The answer is application- and implementation-dependent, because it depends on how much discard can effectively be concealed (1%, 0.01% or 2% in the above examples, assuming no other losses). [This text might be sufficient, but jitter buffers are usually adaptive, and my understanding is that they aim to maximize QoE.  At any particular operating point they convert variable delay into a fixed delay and residual loss.  Since QoE typically depends on both of those resulting attributes, I don't think that they generally ignore the former.]   Nonetheless, it would be useful to settle on a single industry-wide percentile to characterize delay, even if it isn't perfect in every case.
```

Comment:
FWIW I think you are either misunderstanding my comment or we have a very different view as to how this works, because what I wrote *is* exactly the point.  I don't agree with your statement that an implementation has to ensure a certain percentage of packets arrive within the deadline.  In this context, I think QoE is a function of both factors, e.g. QoE = Qmax - k1*latency - k2*log(loss) (this is a simplification of the Cole & Rosenbluth formula for VoIP). 

The network provides a certain delay distribution (likely isn't stationary, but let's assume it is).

The job of the adaptive jitter buffer is to tune to the operating point that produces the fixed latency and residual loss that maximizes the QoE function.

Consider the case where the network provides a P95 latency of 50ms and a P96 latency of 200ms.  In that case, the optimal point might be 50ms and 5% loss.

Or, if the network provides P99 latency of 60ms and P99.999 latency of 62ms, the optimal point might be 62ms and 0.00001% loss. 

So, even for a specific application, you can't simply say that the metric should match the residual loss target of its jitter buffer, because I don't think there is a set loss target.  But, I still think that something like a P99 metric is useful.

Reply:
Yes, in theory there is an optimization. But surely the slope of the delay CDF is so steep that it's hardly a 2-variable optimization. For instance, with 2nd gen AQMs (PIE, FQ-CoDel) the slope of the delay-percentile CDF is 5-10 ms / decade of loss, and with 3rd gen (L4S) it's <1ms / decade. So surely, pragmatically, it's solely about increasing the de-jitter buffer enough to reach a discard percentage that is low enough to be easily concealable... whatever delay that takes. 

I don't know whether you're only going on the comment stream, but I had also changed the text to give a nod to a trade-off with delay. I didn't really want to because it muddies the rationale for choosing a particular percentile. But I think it's OK (and you'd already said "the text might be sufficient" earlier). I think we should now both understand each other's positions, so we can now just focus on the text - if it's not acceptable, pls say.